pyart/io/nexrad_level2.py

```diff
@@ -426,7 +434,10 @@ def get_target_angles(self, scans=None):
         if scans is None:
             scans = range(self.nscans)
         if self._msg_type == '31':
-            cut_parameters = self.vcp['cut_parameters']
+            if self.vcp is not None:
+                cut_parameters = self.vcp['cut_parameters']
+            else:
+                cut_parameters = [{'elevation_angle' : 0.0}]*self.nscans
```

Comment:
for pep8 and consistency I would change remove the space before the colon and added spaces around the multiplication:
```
cut_parameters = [{'elevation_angle': 0.0}] * self.nscans
```